modules/insights-operator-configuring-sca.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/insights-operator-simple-access.adoc
+
+
+[id="insights-operator-configuring-sca_{context}"]
+= Configuring Simple Content Access import interval
+
+You can configure how often the Insights Operator imports the RHEL Simple Content Access (SCA) certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you may want to shorten this interval if you update your SCA configuration in Red Hat Subscription Management.
+
+This procedure describes how to update the import interval to one hour. 
+
+.Prerequisites
+
+* You are logged in to {product-title} web console as `cluster-admin`.
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Search for the *support* secret using the *Search by name* field. If it does not exist, click *Create* -> *Key/value secret* to create it.
+. Click the *Options* menu {kebab}, and then click *Edit Secret*.
+. Click *Add Key/Value*.
+. Create a key named `ocmInterval` with a value of `1h`, and click *Save*.
++
+[NOTE]
+====
+The interval `1h` can also be entered as `60m` for 60 minutes. 
```

Comment:
We should point out what the default value is

Reply:
It's mentioned there:
>The certificate import normally occurs every 8 hours